packages/app-project/stores/Organization.js

```diff
@@ -5,9 +5,11 @@ import numberString from './types/numberString'
 const Organization = types
   .model('Organization', {
     id: types.maybeNull(numberString),
+    display_name: types.optional(types.string, ''),
```

Comment:
Do we use display names? The tests expect them but they've been replaced by translation strings.

Reply:
Yeah, they only seem to be here so that the tests pass.